### PR TITLE
chore(Field.PostalCodeAndCity): add mask to postal code to enforce max length of four

### PR DESCRIPTION
**Pull request by joakbjerk**
solves https://github.com/dnbexperience/eufemia/issues/3090

### CODE REVIEW

File: packages/dnb-eufemia/src/extensions/forms/Field/PostalCodeAndCity/__tests__/PostalCodeAndCity.test.tsx [hunk 1]
```diff
@@ -1,10 +1,27 @@
...
+
+    const postalCodeInput = document.querySelector(
+      '.dnb-forms-field-postal-code-and-city__postal-code .dnb-input__input'
+    ) as HTMLInputElement
+
+    expect(postalCodeInput).toHaveValue('')
+
+    await userEvent.type(postalCodeInput, '123456')
+
+    expect(postalCodeInput).not.toHaveValue('123456')
```
Comment (tujoworker) [inline packages/dnb-eufemia/src/extensions/forms/Field/PostalCodeAndCity/__tests__/PostalCodeAndCity.test.tsx:24]:
```suggestion
```

I dont think it makes it "saver" or better "readable". 

Let me know what do you think.

**Review by tujoworker — APPROVED** [review]
We may add later a prop to support other countries than Norway. It could be a prop to link paths: countryPath

File: packages/dnb-eufemia/src/extensions/forms/Field/PostalCodeAndCity/__tests__/PostalCodeAndCity.test.tsx (hunk 1 above)
Comment (joakbjerk) [reply packages/dnb-eufemia/src/extensions/forms/Field/PostalCodeAndCity/__tests__/PostalCodeAndCity.test.tsx:24]: I agree 😄

**Pull request merged**